images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -284,7 +284,7 @@ function Get-GHCVersion {
 }
 
 function Get-GHCupVersion {
-    $(ghcup --version) -match "version v(?<version>\d+(\.\d+){2,})" | Out-Null
+    $(ghcup --version) -match "version (?<version>\d+(\.\d+){2,})" | Out-Null
```

Comment:
https://github.visualstudio.com/virtual-environments/_build/results?buildId=125094&view=logs&j=80d86624-2c9a-5cad-9434-144b838e69fb&t=0cc083f2-e4b4-5c2b-a1d7-966043ae867c

v - is still there.

![image](https://user-images.githubusercontent.com/47745270/166203423-6440c594-4482-4cd7-a61c-cce7d7a16ed8.png)